packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -11,14 +11,23 @@ import {
 	SelectControl,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+const { DOMParser } = window;
 
 /**
  * Internal dependencies
  */
 import CreateMenuArea from './create-menu-area';
 import NavigationEditor from '../navigation-editor';
 
+const noticeId = 'edit-navigation-delete-menu-error';
```

Comment:
What if there are two instances of this component?

Reply:
This component is the entire editor, hardly worth optimizing for two instances. The create menu, for the same reason, does not assume multiple instance.

Reply:
React itself can break this in future updates like Concurrent mode.

Reply:
made it unique in the same fashion media replace flow handles this